dist/demo-app/README.md

```diff
@@ -1,74 +1,27 @@
 # Capital Framework
 
-A Capital Framework demo project. Use grunt to process the source files in `src/` and output them to `dist/`:
+A Capital Framework sample project.
+To preview it, run `python -m SimpleHTTPServer` and open http://localhost:8000 in a web browser.
```

Comment:
That won't work because index.html uses absolute paths for the stylesheets... which we probably shouldn't be doing. I'll switch them to relative URLs.